sdk-extensions/incubator/build.gradle.kts

```diff
@@ -22,12 +27,63 @@ dependencies {
   implementation(project(":sdk-extensions:autoconfigure-spi"))
   implementation("org.snakeyaml:snakeyaml-engine")
 
-  // io.opentelemetry.sdk.extension.trace.zpages
+  // io.opentelemetry.sdk.extension.incubator.trace.zpages
   implementation(project(":semconv"))
   compileOnly("com.sun.net.httpserver:http")
 
+  // io.opentelemetry.sdk.extension.incubator.fileconfig
+  implementation("com.fasterxml.jackson.core:jackson-databind")
+  implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml")
```

Comment:
Will this eventually make jackson a hard dependency for the sdk or just this extension?

Reply:
`jackson-databind` gets CVEs very often, I'd sleep better if we don't include it in the SDK (at some point); would `jackson-jr` be able to handle the generated POJOs?

Reply:
Considering this is for configuration, which is likely only done at startup I feel we should go with a potentially less performant option and not use jackson in exchange for fewer dependencies and CVE exposure.

Reply:
It's less of the performance I'm worried about and more the additional complexity to manually do the parsing that jackson does automatically. Additionally, there is likely to be a fair amount of churn in the early versions of the configuration schema - I'm happy enough writing manual parsing code once, but won't sign up to rewrite the manual parsing code for each of the early iterations.